### ISSUE TITLE
WISC link in Tutorial is broken

### ISSUE THREAD

**Issue by thomini** [CLIMADA-project/climada_python#610]
The WISC link stated in the Tutorial page https://climada-python.readthedocs.io/en/stable/tutorial/climada_hazard_StormEurope.html#Reading-Data is broken, also I can not find the storm footprints for Europe in the web.

Thank you!

**Comment by peanutfun** [comment]
The broken link is https://wisc.climate.copernicus.eu/wisc

The same link is given on this overview page: https://climate.copernicus.eu/windstorm-information-service So it might be possible that this is a temporal issue with the website.

**Comment by thomini** [comment]
Yes, true, I also sent a request to ECMWF - I will let you know in case there is an update. Thank you!!

**Comment by thomini** [comment]
Good morning! ECMWF replied the following:

>"The WISC Portal is indeed no longer operational. The data is however available to download from the Climate Data Store (CDS):
https://cds.climate.copernicus.eu/cdsapp#!/dataset/sis-european-wind-storm-indicators?tab=overview  "

However, I think there might be differences in the data because footprints downloaded form this link are roughly 37MB per footprint, whereas the previous data size was roughly 2MB per footprint.

**Comment by peanutfun** [comment]
Thanks for reporting back! Will you give it a try with the "new" data? If the functions presented in the tutorial still work out, the increase in data size is not an issue, I think.

**Comment by thomini** [comment]
Sure, I will give it a try in the following days, and I will let you know the outcome!

**Comment by thomini** [comment]
I have tried to read one file only:
```python
from climada.hazard import StormEurope

storm_instance = StormEurope()
storm_instance.from_footprints('C3S_fp_era5_sd_2020020721_115_Ciara1_v1.nc')
```
but unfortunately the Kernel crashed after around 15 minutes:

```log
Canceled future for execute_request message before replies were done
The Kernel crashed while executing code in the the current cell or a previous cell. Please review the code in the cell(s) to identify a possible cause of the failure. Click [here](https://aka.ms/vscodeJupyterKernelCrash) for more info. View Jupyter [log](command:jupyter.viewOutput) for further details.
```
I tried also with the deprecated `.read_footprints`, but with the same result.

So probably the size is an issue on my machine. What do you think?

**Comment by peanutfun** [comment]
Could you upload your jupyter notebook and the file here: https://polybox.ethz.ch/index.php/s/ODnHI1SE9AROhox ?

It might be a large amount of data but it should not take 15 min to load it. Also, I will have to see why/where the kernel crashed. I can give it a try myself. How/where did you download the file?

**Comment by thomini** [comment]
Sure, I have put the files to the folder.

Data download:
* I went to https://cds.climate.copernicus.eu/cdsapp#!/dataset/sis-european-wind-storm-indicators?tab=form
and choose "Windstorm footprints" for the years 2017, 2020 and 2021.
* In this download the file used in the code is included

Thank you for checking!

**Comment by peanutfun** [comment]
I can replicate your problem with the crashing kernel. For me, the crash happens much faster and it is due to a memory overflow. It occurs at [climada/hazard/centroids/centr.py#L822](https://github.com/CLIMADA-project/climada_python/blob/main/climada/hazard/centroids/centr.py#L822), so it is probably related to the exact coordinates of the dataset, and *not* the CLIMADA code itself.

The dataset contains about 8 million coordinates. I am not sure if this is too much to handle for computing a buffer envelope, but it seems to me like it should not. Does somebody have experience with that, maybe @tovogt or @ThomasRoosli? Script and file here: https://polybox.ethz.ch/index.php/s/ODnHI1SE9AROhox

@thomini Note that I updated your script. `StormEurope.from_footprints` is a so-called `classmethod` that is intended to create an instance of `StormEurope` and return it. You can call it from an existing instance (in which case the existing instance has no effect whatsovever), but you must definitely keep its return value, because the new instance will be discarded otherwise. See https://stackoverflow.com/a/12179752 for a nice example and use-case.

**Comment by thomini** [comment]
@peanutfun Thank you for the comment!!

**Comment by GabrielFo** [comment]
I have the same problem and cannot load European Storm footprints downloaded on Copernicus website.
After around 10minutes, I get the error message below.
I'm working on a 300go RAM machine, so it's not a memory issue.
I think it might be related to the Storm Severity Index, which is present in synthetic Storm Dataset but not in real event datasets. 
Maybe there should be some try / except to add in the _read_one_nc method of from_footprints, like if ssi is not present, then set it to None.

storm_instance = StormEurope.from_footprints(DATA_PATH)
  xy_pixels = self.geometry.buffer(res / 2).envelope
Traceback (most recent call last):

  Cell In[23], line 1
    storm_instance = StormEurope.from_footprints(DATA_PATH)

  File ~\AppData\Local\anaconda3\envs\climada_env\lib\site-packages\climada\hazard\storm_europe.py:187 in from_footprints
    hazard_list = [

  File ~\AppData\Local\anaconda3\envs\climada_env\lib\site-packages\climada\hazard\storm_europe.py:188 in <listcomp>
    cls._read_one_nc(

  File ~\AppData\Local\anaconda3\envs\climada_env\lib\site-packages\climada\hazard\storm_europe.py:248 in _read_one_nc
    ssi_wisc = np.array([float(ncdf.ssi)])

  File ~\AppData\Local\anaconda3\envs\climada_env\lib\site-packages\xarray\core\common.py:278 in __getattr__
    raise AttributeError(

AttributeError: 'Dataset' object has no attribute 'ssi'

**Comment by emanuel-schmid** [comment]
Thanks, @GabrielFo. The European Storm footprints downloaded on Copernicus website - could you perhaps provide some more details about them? Where were they downloaded from (url) and what was their specification?

**Comment by GabrielFo** [comment]
Sure, the url is this one : https://cds.climate.copernicus.eu/cdsapp#!/dataset/sis-european-wind-storm-indicators?tab=form.
I'm downloading Windstorm tracks and windstorm footprints indicator.
Bug reported above where related to C3S_fp_era5_sd_2020020721_115_Ciara1_v1.nc.

Moreover, I tried to look at some other footprints, namely the Lothar tempest from 26th December 1999 (C3S_fp_era5_sd_19991226_62_0_Lothar_v1.nc) : the file cannot be read because the code in CLIMADA requires 'latitude', 'longitude' and 'max_wind_gust' as attributes, which are not named the exact same way in this file (namely 'Latitude', 'Longitude', 'FX').

**Comment by peanutfun** [comment]
@thomini @GabrielFo It might be worth checking out the `storm_europe` datasets provided from the Climada Data API. They are based on the "old-style" WISC data with historical events from years 1940 to 2014: https://climada.ethz.ch/data-types/

**Comment by peanutfun** [comment]
Thanks, @thomini. We'll keep it open to track all issues that accumulated for the `StormEurope` module. At the moment, we have to consider it broken, see https://github.com/CLIMADA-project/climada_python/discussions/811

**Comment by peanutfun** [comment]
From #885:

> Could this be updated if the data is still available? Is it [this ](https://cds.climate.copernicus.eu/cdsapp#!/dataset/sis-european-wind-storm-indicators?tab=overview)the data?
